code/modules/events/sentience.dm

```diff
@@ -37,7 +37,8 @@
 	SA.health = SA.maxHealth
 	SA.del_on_death = FALSE
 	greet_sentient(SA)
-	print_command_report(sentience_report, "[command_name()] Update")
+	print_command_report(sentience_report, "[using_map.dock_name] Update")
+	command_announcer.autosay("A classified message has been printed out at all communication consoles.")
```

Comment:
Randomly changing event balance. Nice.

Reply:
How is this changing event balance?

Reply:
This shouldn't have been changed in this PR, period. It's not appropriate to adjust the way events are announced.